code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -203,22 +203,25 @@
 	wield_delay = 16
 	aim_slowdown = SLOWDOWN_ADS_SPECIALIST
 	var/datum/ammo/ammo_secondary = /datum/ammo/bullet/smartgun/lethal//Toggled ammo type
-	var/shells_fired_max = 20 //Smartgun only; once you fire # of shells, it will attempt to reload automatically. If you start the reload, the counter resets.
+	var/shells_fired_max = 50 //Smartgun only; once you fire # of shells, it will attempt to reload automatically. If you start the reload, the counter resets.
 	var/shells_fired_now = 0 //The actual counter used. shells_fired_max is what it is compared to.
 	var/restriction_toggled = 1 //Begin with the safety on.
 	gun_skill_category = GUN_SKILL_SMARTGUN
 	attachable_allowed = list(
+						/obj/item/attachable/extended_barrel,
 						/obj/item/attachable/heavy_barrel,
+						/obj/item/attachable/flashlight,
+						/obj/item/attachable/lasersight,
```

Comment:
These two offer no trade-off nor are thematically fitting. Don't add stuff just to add it.

Reply:
The tradeoff with the laser sight is that you don't get to run a BFA which can actually worth a shit now that the EB is a thing in exchange for a minor accuracy bonus.

Also a rail light is totally thematic, and the benefit is minor given your thermals.